app/src/main/java/com/asfoundation/wallet/topup/payment/AdyenTopUpFragment.kt

```diff
@@ -265,10 +278,23 @@ class AdyenTopUpFragment : DaggerFragment(), AdyenTopUpView {
 
   override fun topUpButtonClicked() = RxView.clicks(button)
 
+  override fun billingAddressInput(): Observable<Boolean> {
+    return billingAddressInput!!
+  }
+
+  override fun retrieveBillingAddressData(): Observable<BillingAddressWrapper> {
+    return Observable.just(billingAddressWrapper)
+  }
+
```

Comment:
In the payment fragment it's a single, but do we need it to rx? Without Rx we can avoid using the wrapper